platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/SecureCxfClientFactory.java

```diff
@@ -97,6 +98,10 @@ public SecureCxfClientFactory(String endpointUrl, Class<T> interfaceClass, List<
                     "Called without a valid URL, will not be able to connect.");
         }
 
+        if (endpointUrl.contains("${") && endpointUrl.contains("}")) {
```

Comment:
I believe this check is unnecessary. Calling it with no properties in the string is effectively a noop.